packages/gatsby-source-filesystem/package.json

```diff
@@ -10,15 +10,14 @@
     "@babel/runtime": "^7.11.2",
     "better-queue": "^3.8.10",
     "chokidar": "^3.4.2",
-    "file-type": "^12.4.2",
+    "file-type": "^16.0.0",
```

Comment:
You need to go and find all other references to `fileType` in this package as the API changed (see tests). Run `yarn test` to confirm. In particular in gatsby-source-filesystem/src/create-file-node-from-buffer.js and gatsby-source-filesystem/src/create-remote-file-node.js

Reply:
@pvdz Thanks, will fix!